## Welcome to Cloud-SPAN on GitHub! üëã

Cloud-SPAN develops training in **Sp**ecialised **an**alyses for environmental 'omics with Cloud-based High Performance Computing. The training is aimed at both Environmental Biotechnology researchers and the Research Computing teams that support them.

The project has been running since September 2021 and is collaboration between the [Department of Biology](https://www.york.ac.uk/biology/) at the University of York, UK, and the [Software Sustainability Institute](https://www.software.ac.uk/), and funded by the [UKRI innovation scholars award](https://www.ukri.org/news/initiatives-boost-health-and-bioscience-skills-and-industry/) under project reference [MR/V038680/1](https://gtr.ukri.org/projects?ref=MR%2FV038680%2F1).

### Our modules

We have currently developed three modules:

| Module                     | Description |
| -------------------------- | ---------|
| [Prenomics](https://cloud-span.github.io/prenomics00-intro/)| Prenomics is a 4 - 6 hour module that teaches the basics of command-line programming, including: (1) file directory structure, (2) use of command-line utilities to connect to and use cloud computing and storage resources and (3) basic shell commands for file navigation and basic script writing. It is designed to prepare people for  [Genomics](https://cloud-span.github.io/00genomics) but, depending on previous experience, you may not need it. There is short (~5 minutes) [Self-assessment Quiz](https://shiny.york.ac.uk/er13/prenomics-quiz/#section-why) to help you decide if you would benefit from attending Prenomics before the Genomics.|
| [Genomics](https://cloud-span.github.io/00genomics/) | Genomics is a 8 - 12 hour module that teaches data management and analysis for genomics research including: (1) best practices for organization of bioinformatics projects and data, (2) use of command-line utilities to connect to and use cloud computing and storage resources, (3) use of command-line tools for data preparation, (4) use of command-line tools to analyze sequence quality and perform and automate variant calling. |
| [Create Your Own AWS Instance](https://cloud-span.github.io/create-aws-instance-0-overview/)| A 2 hour self-study module to teach you how to create and manage your own Cloud-SPAN Amazon Web Services (AWS) instance just like the one used in the Cloud-SPAN courses Prenomics and Genomics. If you attend tutor-led editions of Cloud-SPAN‚Äôs Prenomics and Genomics courses you do not need to create your own instance. We will do that for you! But if would like to practice afterwards, or study the courses in your own time, you will need to create an instance first.

The following modules are in development:
-  Metagenomcs
-  Statistically useful experimental design
-  Scripting and automation
-  Cloud Administration for training delivery

Our aim is to make our materials FAIR - **F**indable, **A**ccessible, **I**nteroperable and **R**eusable.
![FAIR-for-training-materials png](https://user-images.githubusercontent.com/7593411/169241689-6c0fe466-8712-4fb7-b9d7-2c7f45755e75.png)
Illustration from Luc Wiegers and Celia van Gelder: [https://doi.org/10.5281/zenodo.3593257](https://doi.org/10.5281/zenodo.3593257).
[https://doi.org/10.1371/journal.pcbi.1007854.g001](https://doi.org/10.1371/journal.pcbi.1007854.g001)

## Our Handbook
The Cloud-SPAN team are dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. We hope aim to develop a community of practice around our materials.
We have a [Handbook](https://cloud-span.github.io/CloudSPAN-handbook/) that gives 
‚≠ê An introduction to the project
ü§ù Our Code of Conduct
üéì More information on our Courses
üë™ An open invitation to the Cloud-SPAN Community
üìå Information about the FAIR Principles
üìúCloud-SPAN Online Forum


