## Welcome to Cloud-SPAN on GitHub! üëã

Cloud-SPAN develops training in 'omics with Cloud-based High Performance Computing. The training is aimed at both biscience researchers and the Research Computing teams that support them.

The project has been running since September 2021 and is collaboration between the [Department of Biology](https://www.york.ac.uk/biology/) at the University of York, UK, and the [Software Sustainability Institute](https://www.software.ac.uk/), and funded by the [UKRI innovation scholars award](https://www.ukri.org/news/initiatives-boost-health-and-bioscience-skills-and-industry/) ([MR/V038680/1](https://gtr.ukri.org/projects?ref=MR%2FV038680%2F1)) the Natural Environment Research Council ([NE/X006999/1](https://gtr.ukri.org/projects?ref=NE%2FX006999%2F1#/tabOverview) and  ([NE/Y003527/1](https://gotw.nerc.ac.uk/list_full.asp?pcode=NE%2FY003527%2F1))). 

### Our modules

| Module                     | Description |
| -------------------------- | ---------|
| [Prenomics](https://cloud-span.github.io/prenomics00-intro/)| Prenomics is a 4 - 6 hour module that teaches the basics of command-line programming, including: (1) file directory structure, (2) use of command-line utilities to connect to and use cloud computing and storage resources and (3) basic shell commands for file navigation and basic script writing. It is designed to prepare people for  [Genomics](https://cloud-span.github.io/00genomics) but, depending on previous experience, you may not need it. There is short (~5 minutes) [Self-assessment Quiz](https://shiny.york.ac.uk/er13/prenomics-quiz/#section-why) to help you decide if you would benefit from attending Prenomics before the Genomics.|
| [Genomics](https://cloud-span.github.io/00genomics/) | Genomics is a 8 - 12 hour module that teaches data management and analysis for genomics research including: (1) best practices for organization of bioinformatics projects and data, (2) use of command-line utilities to connect to and use cloud computing and storage resources, (3) use of command-line tools for data preparation, (4) use of command-line tools to analyze sequence quality and perform and automate variant calling. |
| [Create Your Own AWS Instance](https://cloud-span.github.io/create-aws-instance-0-overview/)| A 2 hour self-study module to teach you how to create and manage your own Cloud-SPAN Amazon Web Services (AWS) instance just like the one used in the Cloud-SPAN courses Prenomics and Genomics. If you attend tutor-led editions of Cloud-SPAN‚Äôs Prenomics and Genomics courses you do not need to create your own instance. We will do that for you! But if would like to practice afterwards, or study the courses in your own time, you will need to create an instance first.
| [Statistically useful experimental design](https://cloud-span.github.io/experimental_design00-overview/)| 2 - 3 hour workshop about designing ‚Äòomics experiments including platform choice, replication and controls, sequence coverage and depth and multiple testing corrections. The module does not require any software or coding. Some principles of design will be presented followed by discussion of their application using case studies including the participants's own designs. We assume no experience with designing omics‚Äô experiments but some previous experience of experimental design and statistical analysis - such as would be covered in an undergraduate bioscience degreee - would be useful.
| [Metagenomics](https://cloud-span.github.io/metagenomics00-overview/)| This course teaches data analysis for metagenomics projects. It covers how to (1) generate and QC a metagenome assembly, (2) ‚Äòbin‚Äô the assembly into metagenome assembled genomes (MAGs) also known as bins, (3) identify the taxonomy of these MAGs and, (4) calculate diversity metrics and add functional annotation to identify the products of genes identified in the assembled MAGs. There is 4 - 8 hrs teaching material but the course is delivered over 3 or 4 weeks since many of the analyses take several hours to run. Each week there is a taught session covering the background to the week's material before you work though the lesson at your own pace, followed by a drop-in session to help with any problems
| [Metagenomics for Environmental Scientists](https://cloud-span.github.io/nerc-metagenomics00-overview/)| This course is aimed at environmental scientists with little or no experience of using high performance computing (HPC) for data analysis. It is taught over two-weeks online using a mixture of live coding, online lectures, offline time for long analyses to complete and drop-in sessions. It covers (1) Using the command line to log into cloud resources, navigate filesystems and carry out filesystem housekeeping, (2) what metagenomics is, the difference between genomics and metagenomics and the different types of sequencing platforms, and (3) metagenomics analysis including quality control, assembly, polishing, binning and taxonomic assignment
| [Automated Management of AWS Instancess](https://cloud-span.github.io/cloud-admin-guide-0-overview/)| This course teaches how to automatically manage multiple Amazon Web Services (AWS) instances such as might be used for delivering training courses. It uses Bash Shell scripts to create, configure, stop, start and delete one or more instances with a single invocation of a script.
| [Core R](https://github.com/Cloud-SPAN/core-r)| This online two-hour workshop is an introduction to R for complete beginners. It teaches you how to find your way round RStudio, use the basic data types and structures in R and how to organise your work with scripts and projects. It also teaches you how to import data, summarise it and create and format a graph. The workshop assumes no prior experience of coding.


Our aim is to make our materials FAIR - **F**indable, **A**ccessible, **I**nteroperable and **R**eusable.
![FAIR-for-training-materials png](https://user-images.githubusercontent.com/7593411/169241689-6c0fe466-8712-4fb7-b9d7-2c7f45755e75.png)
Illustration from Luc Wiegers and Celia van Gelder: [https://doi.org/10.5281/zenodo.3593257](https://doi.org/10.5281/zenodo.3593257).
[https://doi.org/10.1371/journal.pcbi.1007854.g001](https://doi.org/10.1371/journal.pcbi.1007854.g001)

## Our Handbook
The Cloud-SPAN team are dedicated to providing a welcoming and supportive environment for all people, regardless of background or identity. We hope aim to develop a community of practice around our materials.
We have a [Handbook](https://cloud-span.github.io/CloudSPAN-handbook/) that gives:

‚≠ê An introduction to the project  
ü§ù Our Code of Conduct  
üéì More information on our Courses  
üë™ An open invitation to the Cloud-SPAN Community  
üìå Information about the FAIR Principles  
üìúCloud-SPAN Online Forum  


